pbr/config/scene_config.py

```diff
@@ -44,6 +45,60 @@
         "kinematics_variance": 0.5,  ## Determines how much variance the random poses will have
         "mask": {"index": 3, "colour": (0, 0, 1, 1)},
     },
+    "misc_robot": {
+        "robot_list": {
+            "darwin": {
+                "mesh_path": path.abspath(path.join(res_path, "robot", "darwin_fixed.fbx")),
```

Comment:
Only a minor suggestion, but are we keeping these as "..._fixed" or should we change them to only be the name of the robot

Reply:
I've standardized all the robot names for the new resources folder. Just haven't pushed that yet.